src/status_im2/contexts/syncing/scan_sync_code/view.cljs

```diff
@@ -434,5 +403,14 @@
              flashlight-icon])]]))))
 
 (defn view
-  [props]
-  [:f> f-view props])
+  [_props]
+  (let [qr-code-succeed? (reagent/atom false)]
+    (navigation.util/create-class-and-bind
+     "sign-in-intro"
```

Comment:
I believe the name should be received in a prop because we use this view (as the screen) in two places:

1. Sign In
2. Scan sync code

Reply:
Should we match it with the name declared in `screens.cljs`?
https://github.com/status-im/status-mobile/blob/eb3de978cef417673fabe21eb1c60fc72f905e27/src/status_im2/navigation/screens.cljs#L210

Reply:
Done, the `screen-name` is received as a parameter